pkg/services/ngalert/notifier/channels/opsgenie.go

```diff
@@ -180,13 +189,25 @@ func (on *OpsgenieNotifier) buildOpsgenieMessage(ctx context.Context, alerts mod
 	var tmplErr error
 	tmpl, data := TmplText(ctx, on.tmpl, as, on.log, &tmplErr)
 
-	title := tmpl(DefaultMessageTitleEmbed)
-	description := fmt.Sprintf(
-		"%s\n%s\n\n%s",
-		tmpl(DefaultMessageTitleEmbed),
-		ruleURL,
-		tmpl(`{{ template "default.message" . }}`),
-	)
+	titleTmpl := on.Message
+	if strings.TrimSpace(titleTmpl) == "" {
+		titleTmpl = `{{ template "default.title" . }}`
+	}
+
+	title := tmpl(titleTmpl)
+	if len(title) > 130 {
+		title = title[:130]
```

Comment:
I would suggest following in pagerduty's footsteps here and adding an indicator that the title was truncated.
```suggestion
		title = title[:127] = "..."
```

Reply:
Ok, I'll have to modify the tests too!